app-web/src/pages/contentContribution.js

```diff
@@ -0,0 +1,50 @@
+import { React, useState } from 'react';
+import Layout from '../hoc/Layout';
+
+// components
+import { SEO } from '../components/SEO/SEO';
+import { Title } from '../components/Page';
+import Main from '../components/Page/Main';
+import { SelectDropdown } from '../components/Form/form';
+import CurrentForm from '../components/Form/CurrentForm';
+import TopicForm from '../components/Form/TopicForm';
+import JourneyForm from '../components/Form/JourneyForm';
+
+const contentContribution = () => {
+  const [currentVal, setVal] = useState('');
```

Comment:
should be called currentVal, setCurrentVal 

also these are terrible variable names :P